backend/src/libs/packages/config/config.ts

```diff
@@ -1,8 +1,11 @@
 import { logger } from '~/libs/packages/logger/logger.js';
 
 import { Config } from './config.package.js';
+import { TokenConfig } from './token.config.package.js';
 
 const config = new Config(logger);
+const tokenConfig = new TokenConfig(logger);
```

Comment:
```suggestion
```
you don't need an additional class. just add a new getter `AUTH` to the config class. without ENV parsing for them. just hardcode them